packages/lisk-transactions/src/transactions/helpers/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
whats the difference between utils and helpers?

Reply:
The existing util functions such as `sign_and_verify/verifyTransaction` and `utils/get_transaction_id` uses `getTransactionBytes` which uses getAssetBytes.  As per previous discussion this may negate the point of classes by doing this. If we are to stick to handling assets in each class, we need new functions to fit this approach.  How about `helpers/verifySignature(key, signature, txBytes, isSecondSignature)` where `txBytes` will come from each type classes' `getBytes` method (and remove the `helpers/getBytes` and put back previous `getBasicBytes` method which will be used in `getBytes` method).

Reply:
helpers and utils are essentially the same imo.
We can name function differently, and put deprecated comments in one of them if it will not be used in the future.
as per similar functions, i think we can put it in the same file in the utils folder.
maybe in the base class we can have
```
getBytes => (return whole bytes including singature/signSignature) uses abstract getAssetBytes
getBasicBytes => (return bytes without signature) uses abstract getAssetBytes
abstract getAssetBytes as it is now
```
I would prefer not to use `skip` flag.
also `verifySignature(publicKey, signature, txBytes)` looks good. and depends on where it's called `txBytes` changes